src/templates/components/textarea.html

```diff
@@ -3,6 +3,7 @@
   ng-model="data[component.key]"
   ng-disabled="readOnly"
   ng-required="isRequired(component)"
+  spellcheck="{{ component.spellcheck }}"
```

Comment:
We need to separate out this into a different pull request so that it does not get hung up in the review process.